Zotero/EventObservableWindow.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  EventObservableWindow.swift
+//  Zotero
+//
+//  Created by Michal Rentka on 20.11.2024.
+//  Copyright © 2024 Corporation for Digital Scholarship. All rights reserved.
+//
+
+import UIKit
+
+final class EventObservableWindow: UIWindow {
+    override func sendEvent(_ event: UIEvent) {
+        super.sendEvent(event)
+        guard !(event.allTouches ?? []).isEmpty else { return }
```

Comment:
Are other events that we should consider, that may be applicable here, like device rotation?

Reply:
Added in d4b36edc9e401ac3c1b89c67f1705ab1898c2593